### ISSUE TITLE
Deprecated warning on PHP 8.1

### ISSUE THREAD

**Issue by webmastervinay** [laravel/valet#1109]
## Problem description
Whenever you run any valet command, it shows below error:
```
Deprecated: Return type of Tightenco\Collect\Support\LazyCollection::jsonSerialize()
```

## Expected behavior
This should run without any warning.

## Steps to Reproduce
1. PHP 8.1 Upgrade (I am on MacOS - Monterey 12.0.1)
2. Run any valet command, say valet links and you will get this warning multiple times

## Possible solution
Looks like the package tightenco/collect (dependency of laravel/valet) is not updated for PHP 8.1

**Comment by driesvints** [comment]
Please report this to tightenco/collect because we don't maintain this package.

**Comment by digitall-it** [comment]
Stumbled upon the same problem for Symfony Console, reported at https://github.com/symfony/symfony/issues/42199
`
Deprecated: Return type of Symfony\Component\Console\Helper\HelperSet::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /Users/[...]/.composer/vendor/symfony/console/Helper/HelperSet.php on line 104
`

**Comment by digitall-it** [comment]
Ok I reported to symfony/console here https://github.com/symfony/symfony/issues/42199
seems like they fixed the issue. This does not seem to be a local problem of mine, as I did perform those operations

- wiped up my homebrew's php version and reinstalled it from scratch
- wiped out my laravel valet installation and reinstalled it from scratch
- composer global updated everything
- went into global composer directory, deleted the vendor folder and did a composer install
- went into global composer directory, deleted the vendor folder and did a composer update

those operations did not fix the issue.

On the educational side of things, I did dive in Valet's folder to check if that composer.json symfony/console version was updated, but it wasn't referenced. As I know for a fact that it is used (hence the deprecation warning) I wanted to know if there's a composer command to show the dependancy tree of a package, so to know what was using an older version. 
I think that  "symfony/process": "^3.0|^4.0|^5.0|^6.0" is not the culprit, but https://github.com/tighten/collect looks sus!
